### DISCUSSION TITLE
roboflow package for Jetson Nano

### DISCUSSION THREAD

**Discussion by FarStryke21** [roboflow/roboflow-python#141] [Q&A]
I am trying (and failing) to install the roboflow python package on my Jetson Nano. Is the package incompatible with this board? I am using pip3 to install the package. From the looks of it, the install is failing because of the opencv requirement.

**Comment by capjamesg** [comment]
Hello @FarStryke21! Can you share the error you received? We are actively working on addressing issues with the `opencv-python` dependency.
